roles/ceph-osd/templates/ceph-osd.service.j2

```diff
@@ -5,8 +5,10 @@ After=docker.service
 [Service]
 EnvironmentFile=-/etc/environment
 ExecStartPre=-/usr/bin/docker stop {{ ansible_hostname }}-osd-dev%i
-ExecStartPre=-/usr/bin/docker rm -f  {{ ansible_hostname }}-osd-dev%i
-ExecStart=/usr/bin/docker run --rm --net=host --pid=host\
+ExecStartPre=-/usr/bin/docker rm -f {{ ansible_hostname }}-osd-dev%i
+ExecStart=/usr/bin/docker run --rm --net=host \
+   --cap-add SYS_ADMIN \
```

Comment:
I am searching around and can't seem to find much explanation of the pros and cons of using --cap-add SYS_ADMIN vs. using --privileged.  Can you point me to something that explains this?